examples/cors/src/main/java/io/helidon/examples/cors/Main.java

```diff
@@ -65,14 +66,12 @@ public static void main(final String[] args) {
      * Setup routing.
      *
      * @param routing routing builder
-     * @param config  configuration of this server
      */
-    static void routing(HttpRouting.Builder routing, Config config) {
-
-        GreetService greetService = new GreetService(config);
+    static void routing(HttpRouting.Builder routing) {
+        Config config = Config.global();
 
         // Note: Add the CORS routing *before* registering the GreetService routing.
-        routing.register("/greet", corsSupportForGreeting(config), greetService)
+        routing.register("/greet", corsSupportForGreeting(config), new GreetService())
                 .addFeature(ObserveFeature.create());
```

Comment:
As written, this code fetches the global config and passes it to `corsSupportForGreeting` whereas the greet service  invokes `Config.global()` itself to get config. Just to emphasize this new division of labor, how about if `corsSupportForGreeting` also fetched `Config.global()` for itself, rather than having this code fetch it and pass the config as a parameter?

Reply:
Done